### ISSUE TITLE
Missing Vox Box Megaissue

### ISSUE THREAD

**Issue by Flareguy** [space-wizards/space-station-14#33686]
## Description
Goal: Give all of these stations a Vox Break Room. It's supposed to be standard issue, as it allows voxes to eat and drink safely without having to use something like a syringe.
- [ ] Marathon
- [ ] Meta
- [ ] Fland
- [ ] Packed
- [ ] Omega
- [ ] Reach (may not be possible or reasonable due to this station's miniscule size)

**General guidelines for mapping them**
- Try to make them simplistic in nature, they're utilitarian in use (and sometimes implied to be generally neglected.)
- Windoors are generally recommended for the cycling mechanism, since they're compact and fit the utilitarian nature of the vox box
- Put them in public "off the beat and path" areas like dorms, evac, or arrivals if possible
- Not recommended, but if you have nowhere to else to put them, shunting them in to maints will work. People can find them using the map beacon.

**Comment by UbaserB** [comment]
Cog already has a vox box

**Comment by Flareguy** [comment]
> Cog already has a vox box

Whoops. I'll remove that one from the list.

**Comment by PJB3005** [comment]
I'm going to just preemptively close this issue. Vox Boxes are a terrible game mechanic that I have discussed months ago as something we should not have. Not to mention this part:

> Try to make them simplistic in nature, they're utilitarian in use (and sometimes implied to be generally neglected.)

This is literally codifying speciesism in the game design and I fucking appalled to see that we even have some Vox Boxes like this in the game, let alone some of these being added **after** I had to put everybody on blast for the last "Racist Vox implications" thing.
